### PR TITLE
Document Rocky Linux 8 as a client

### PR DESCRIPTION
**Pull request by juliogonzalez**
# Description

Document Rocky Linux 8 as a client

I am documenting it for SUSE Manager as well at master, since it will be part of 4.3. And this will need a back port to 4.2 soon as well (I will request it later).

# Target branches

Which documentation version does this PR apply to?

- [x] Master (Default)
- [x] Manager-4.2 (Later!!!!!)
- [ ] Manager-4.1
- [ ] Manager-4.0

# Links

Fixes https://github.com/SUSE/spacewalk/issues/15322

### CODE REVIEW

**Comment by juliogonzalez** [comment]
PDFs:
[suse_manager_client-configuration_guide.pdf](https://github.com/uyuni-project/uyuni-docs/files/6808226/suse_manager_client-configuration_guide.pdf)
[uyuni_client-configuration_guide.pdf](https://github.com/uyuni-project/uyuni-docs/files/6808227/uyuni_client-configuration_guide.pdf)

File: modules/client-configuration/pages/clients-rocky.adoc [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+Traditional clients are not available on {rocky}{nbsp}8.
+{rocky}{nbsp}8 clients are only supported as Salt clients.
+====
+
+[NOTE]
+====
+Registering {rocky} clients to {productname} is tested with the default SELinux configuration of ``enforcing`` with a ``targeted`` policy.
+You do not need to disable SELinux to register {rocky} clients to {productname}.
+====
+
```
Comment (keichwa) [inline modules/client-configuration/pages/clients-rocky.adoc:17]:
```suggestion



```
We want 3 empty lines before any title ;)

File: modules/client-configuration/pages/clients-rocky.adoc [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+You might notice some disparity in the number of packages available in the AppStream channel between upstream and the {productname} channel.
+You might also see different numbers if you compare the same channel added at a different point in time.
+This is due to the way that {rocky} manages their repositories.
+{rocky} removes older version of packages when a new version is released, while {productname} keeps all of them, regardless of age.
+====
+
+
+include::snippets/appstream_admon.adoc[]
+
+
```
Comment (keichwa) [inline modules/client-configuration/pages/clients-rocky.adoc:92]:
```suggestion


```

**Review by keichwa — APPROVED** [review]
Thanks for documenting this feature!  Very minor proposals:

**Comment by keichwa** [comment]
I forgot to mention: Please add a changelog entry! :)

**Comment by juliogonzalez** [comment]
> I forgot to mention: Please add a changelog entry! :)

I see one, just at the first change of this PR? Or what do you mean?

**Comment by juliogonzalez** [comment]
All suggestions approved.

**Review by keichwa — APPROVED** [review]

File: modules/client-configuration/pages/clients-rocky.adoc [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+ifeval::[{uyuni-content} == true]
+The channels you need for this procedure are:
+
+[[rocky-channels-uyuni-cli]]
+[cols="1,1,1,1", options="header"]
+.CentOS Channels - CLI
+|===
+| OS Version | Base Channel | Client Channel       | AppStream Channel
+| {rocky} 8 | rocky8      | rocky8-uyuni-client | rocky8-appstream
```
Comment (juliogonzalez) [inline modules/client-configuration/pages/clients-rocky.adoc:61]: I just noticed a typo here. I need to fix it. It's `rockylinux` and not `rocky`. Preparing a PR ASAP.

**Pull request merged**